addons/website_event_track/controllers/main.py

```diff
@@ -146,11 +147,12 @@ def event_track_proposal_post(self, event, **post):
             'partner_name': post['partner_name'],
             'partner_email': post['email_from'],
             'partner_phone': post['phone'],
-            'partner_biography': escape(post['biography']),
+            'partner_biography': plaintext2html(post['biography']),
             'event_id': event.id,
             'tag_ids': [(6, 0, tags)],
             'user_id': False,
-            'description': escape(post['description'])
+            'description': plaintext2html(post['description']),
+            'image': base64.b64encode(post['image'].read())
```

Comment:
The tester will probably tell you that as well but it raises an error if you don't select an image (and the field is not mandatory).

Reply:
I didn't get any error when submitting the form without an image